src/util/lang_class.js

```diff
@@ -34,7 +51,13 @@
             })(property);
           }
           else {
-            klass.prototype[property] = source[property];
+            var descriptor = getOwnPropertyDescriptor(source, property);
+
+            if (!descriptor || !(descriptor.get || descriptor.set)) {
+              descriptor = makePropertyDescriptor(source[property]);
+            }
+
+            Object.defineProperty(klass.prototype, property, descriptor);
```

Comment:
Finally, we're going to use the awesome new functionality to set every property using a descriptor.

The first block of code uses the browser's version of `getOwnPropertyDescriptor` if available, but would otherwise use the polyfill.

Then, in the second block, we're going to test the descriptor.
* If there is no descriptor (it is `undefined`);
* Or if there is a descriptor, but there is neither a getter, nor a setter in the descriptor, we can safely assert that the property is meant to be declared as a regular variable as in `source.foo = 'bar'`.

If either of these options are met, we're going to create a descriptor for the property using the default arguments. This would allow other developers to update Fabric without having to re-write all their custom class declarations.

Lastly, in the third block, we're properly defining the property with its intended descriptor.